lib/go-tc/serverchecks.go

```diff
@@ -179,6 +179,21 @@ type GenericServerCheck struct {
 	Checks map[string]*int `json:"checks,omitempty"`
 }
 
+// ServercheckResponseV40 (not to be confused with ServerchecksResponse) is the
+// type of a response from Traffic Ops to a request to its /servercheck
+// endpoint (not to be confused with its /servers/checks endpoint) in version
+// 4.0 of the Traffic Ops API.
+type ServercheckResponseV40 struct {
```

Comment:
This name does _not_ actually collide with anything without that suffix, but it is extremely similar to [`ServerchecksResponse`](https://pkg.go.dev/github.com/apache/trafficcontrol@v5.1.1+incompatible/lib/go-tc#ServerchecksResponse) and I thought the suffix would help tell them apart.

Alternatively, generally our API uses `XsResponse` to represent a response that looks like:
```json
{
    "response": [{"response is an array": "of 'X' objects"}]
}
```
and `XResponse` to represent a response that looks like:
```json
{
    "response": {"response is a single 'X' object": "this is usually used in create/update/delete responses}
}
```
But I named it as a singular just because that matches the endpoint name - should that name have a plural? If so it collides and needs a version suffix (or something)

Reply:
What about `ServerCheckResponse`? That name doesn't appear to be taken yet and has two points of differentiation with `ServerchecksResponse`. Or instead of versions, we could do `ServercheckAPIResponse`. Inserting `API` adds differentiation without tying it to any particular version.